tests/conftest.py

```diff
@@ -55,7 +61,15 @@ def pytest_addoption(parser, pluginmanager):
     parser.addoption("--local_testing_mode", action="store_true", default=False)
     parser.addoption("--enable_cte_optimization", action="store_true", default=False)
     parser.addoption("--skip_sql_count_check", action="store_true", default=False)
-    parser.addoption("--enable-ast", action="store_true", default=False)
+    parser.addoption("--enable_ast", action="store_true", default=False)
+    parser.addoption("--full_ast_validation", action="store_true", default=False)
```

Comment:
nit: maybe a shorter name `--validate-ast`?

Reply:
Do you need to pass both `--enable-ast` and `--full_ast_validation` to run --full_ast_validation or is --full_ast_validation sufficient? I feel we prob. want to add a `help` entry here to document what is required.

Reply:
Yes, you need to pass pytest arguments "--enable_ast --validate_ast" and then run the integ tests.

Noted on the arguments, just following the existing convention here.  The existing code was actually checking for "pytestconfig.getoption("enable_ast")" with underscore so it was previously not working.